src/extensions/forte.py

```diff
@@ -80,6 +80,64 @@ async def cog_command_error(self, ctx, error):
     @commands.group(aliases=["포르테", "ㅍ"], brief="포르테 API 관련 명령어가 모아져 있습니다.")
     async def forte(self, ctx):
         pass
+    @forte.command(aliases=['청약철회'],brief="포르테 아이템 구매를 청약철회 합니다.")
+    async def refund(self, ctx, user: ForteUser):
+        embed=ForteUser.to_embed(user)
+        refund_disabled_env = os.getenv("DISABLE_WITHDRAW_ITEMS").split(",")
+
+        result, resp = await request(
+            "get", f"/users/{user['id']}/items"
+        )
+
+        if resp.status // 100 == 4:
+            message = result.get("message", "Unknown Error")
+            return await ctx.send(f"사용자 아이템 리스트 조회 실패: {message}")
+
+        refundable_items=[]
+        refundable_item_ids=[]
+        for item in result:
+            if str(item['item_id']) not in refund_disabled_env and item['expired'] == 0 and item['consumed'] == 0 and item['sync'] == 0 and item['item']['price'] != 0:
+                refundable_items.append(item)
+                refundable_item_ids.append(str(item['id']))
+        send_text="**ID : 아이템명 : 가격 : 구매일자**\n"
+
+        for item in refundable_items:
+            send_text+=f"`{item['id']}`:`{item['item']['name']}`:`{item['item']['price']}`<:fortepoint:737564157473194014> : `{item['created_at']}`\n"
+        embed.add_field(name="청약철회 가능 아이템",value=send_text,inline=False)
+        if len(refundable_items) == 0:
+            return await ctx.send("청약철회 가능한 아이템이 없습니다.")
+        await ctx.send("청약철회할 아이템 아이디를 입력해주세요.",embed=embed)
+
+        def check(m):
+            return m.author == ctx.author and m.channel == ctx.channel and m.content.isdigit and m.content in refundable_item_ids
+        try:
+            msg = await self.bot.wait_for('message',timeout=10.0, check=check)
+        except asyncio.TimeoutError:
+            return await ctx.send("시간초과로 청약철회를 취소합니다.")
+
+        result, resp = await request(
+            "delete", f"/users/{user['id']}/items/{msg.content}"
+        )
```

Comment:
```suggestion
        await request("delete", f"/users/{user['id']}/items/{msg.content}")
```

리턴값 안쓸거면 괜히 헷갈리지 않게 해두시는게 좋을듯